### ISSUE TITLE
libnsl-1.3.0 has strange config.rpath with invalid shell: config.rpath: line 219: ==: command not found

### ISSUE THREAD

**Issue by trofi** [thkukuk/libnsl#11]
Originally discovered by Agostino Sarubbo in https://bugs.gentoo.org/732100.

There at `./configure` phase shell prints warnings:

```
...
checking if the linker (/usr/x86_64-pc-linux-gnu/bin/ld -m elf_x86_64) is GNU ld... yes
checking for shared library run path origin... /var/tmp/portage/net-libs/libnsl-1.3.0/work/libnsl-1.3.0/config.rpath: line 219: ==: command not found
/var/tmp/portage/net-libs/libnsl-1.3.0/work/libnsl-1.3.0/config.rpath: line 219: ==: command not found
done
checking 32-bit host C ABI... no
...
```

`config.rpath` contains suspicious `if $libdir == /usr/lib | $libdir == /usr/lib64; then` line at:

```
      if $LD --help 2>&1 | grep ': supported targets:.* elf' > /dev/null; then
        if $libdir == /usr/lib | $libdir == /usr/lib64; then
                     hardcode_libdir_flag_spec=""
             fi
      else
        ld_shlibs=no
      fi
```

I'm not sure where that code comes from. `libnsl-1.2.0` does not contain it.

**Comment by thkukuk** [comment]
That's an openSUSE bug, somewhere is a bug report and a fixed gettext should be released in the next days.

**Comment by thkukuk** [comment]
Should be fixed with v2.0.0

**Comment by trofi** [comment]
Works as expected now. Thank you!
